docs/configuration.asciidoc

```diff
@@ -180,6 +180,24 @@ It also expects that a period (`.`) is used to separate the integer and the frac
 | 1.0     | Double 
 |============
 
+[float]
+[[config-transaction-max-spans]]
+==== `TransactionMaxSpans` (performance)
+
+Limits the amount of spans that are recorded per transaction. This is helpful in cases where a transaction creates a very high amount of spans (e.g. thousands of SQL queries). Setting an upper limit will prevent overloading the agent and the APM server with too much work for such edge cases.
```

Comment:
As we discussed at (https://github.com/elastic/apm-agent-dotnet/pull/472#discussion_r324145863), I would recommend adding specification for agent's behavior for the special cases.

Reply:
Documentation was updated